website/common/script/libs/replaceIdsWithTranslations.js

```diff
@@ -0,0 +1,21 @@
+import _ from 'lodash';
+
+/*
+ * Returns a copy of an object where identifiers of translations are
+ * replaced with actual translations in specified language
+ */
+export default function replaceIdsWithTranslations (
```

Comment:
Two things about this function:

- in /common we try to keep only code used both on client and server and since this one is only used on the server, could you move it to website/server/libs/i18n.js?
- I think the name of the function is not very clear, although I don't have any different idea right now

Reply:
@paglias how about localizeData in libs/content.js?